src/main/resources/application-localstack.yml

```diff
@@ -35,7 +35,7 @@ hmpps.sqs:
       queueName: hmpps_prisoner_to_nomis_adjudication_queue
       dlqName: hmpps_prisoner_to_nomis_adjudication_dlq
       subscribeTopicId: domainevents
-      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.damages.updated\", \"adjudication.evidence.updated\", \"adjudication.hearing.created\", \"adjudication.hearing.updated\", \"adjudication.hearing.deleted\" , \"adjudication.hearingCompleted.created\", \"adjudication.hearingCompleted.deleted\", \"adjudication.hearingReferral.created\", \"adjudication.hearingReferral.deleted\", \"adjudication.hearingAdjourn.created\", \"adjudication.hearingAdjourn.deleted\", \"adjudication.punishments.created\", \"adjudication.referral.outcome.prosecution\", \"adjudication.referral.outcome.notProceed\", \"adjudication.referral.outcome.referGov\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.referPolice\", \"adjudication.outcome.notProceed\", \"adjudication.referral.deleted\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.notProceed.deleted\"]}"
+      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.damages.updated\", \"adjudication.evidence.updated\", \"adjudication.hearing.created\", \"adjudication.hearing.updated\", \"adjudication.hearing.deleted\" , \"adjudication.hearingCompleted.created\", \"adjudication.hearingCompleted.deleted\", \"adjudication.hearingReferral.created\", \"adjudication.hearingReferral.deleted\", \"adjudication.hearingAdjourn.created\", \"adjudication.hearingAdjourn.deleted\", \"adjudication.punishments.created\", \"adjudication.punishments.updated\", \"adjudication.referral.outcome.prosecution\", \"adjudication.referral.outcome.notProceed\", \"adjudication.referral.outcome.referGov\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.referPolice\", \"adjudication.outcome.notProceed\", \"adjudication.referral.deleted\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.notProceed.deleted\"]}"
```

Comment:
Wow thats a lot of events! And possibly could do with splitting to multiple lines?

Reply:
Yep, there are still loads more to do!